hack/tools/release/notes.go

```diff
@@ -1,6 +1,3 @@
-//go:build tools
-// +build tools
```

Comment:
Intentional?

Don't ask me why we have that btw :D

Reply:
Not intentional - should be removed now.